### PR TITLE
[JavaScript] Improve Concept Exercise: strings

### PR DESCRIPTION
**Pull request by hayashi-ay**
I removed the reverse task because it relies on a few more advanced features of `array` and this exercise would be prior to `array`. See more detail -> #1298 

We'll introduce `String#trim` instead, so I changed the story accordingly.

I'll add `after.md` in another Issue.

### CODE REVIEW

**Comment by SleeplessByte** [comment]
Looking forward to that next PR 🗡️

**Comment by SleeplessByte** [comment]
image
GitHub seems drunk...

File: languages/javascript/exercises/concept/strings/.docs/instructions.md [hunk 1]
```diff
@@ -91,13 +91,13 @@ backDoorResponse('Stands so high')
 // => "h"
 ```
 
-### 4. Reverse a word
+### 4. Trim a sentence
 
-Implement a function that reverses a word:
+Implement a function that trim a sentence:
```
Comment (tejasbubane) [inline languages/javascript/exercises/concept/strings/.docs/instructions.md:96]:
```suggestion
Implement a function that trims a sentence and returns the last character:
```

Comment (hayashi-ay) [reply languages/javascript/exercises/concept/strings/.docs/instructions.md:96]: Thanks, I'll add this to #1381.

Comment (hayashi-ay) [reply languages/javascript/exercises/concept/strings/.docs/instructions.md:96]: I did.

**Pull request merged**